docs/section11/Rule11-16.md

```diff
@@ -0,0 +1,39 @@
+# ServiceWaterHeating - Rule 11-16
+
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 11-16  
+
+**Rule Description:** "Gas water heaters shall be modeled using natural gas as their fuel.  Exceptions: Where natural gas is not available for the proposed building site, as determined by the rating authority, gas water heaters shall be modeled using propane as their fuel."  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, (h)  
+
+**Evaluation Context:** Each SWH Equipment  
+**Data Lookup:**   
+**Function Call:**
+
+
+**Applicability Checks:**
+- look at each SHW Equipment: `for swh_eq in B_RMD.service_water_heating_equipment:`
+  - if the fuel source is electricity, the rule is not applicable: `if shw_eq.heater_fuel_type == "ELECTRICITY": NOT_APPLICABLE`
+  - otherwise, continue to rule logic: `
+  - `CONTINUE TO RULE LOGIC`
+    ## Rule Logic:
+    - create variable heater_fuel_type: `heater_fuel_type = swh_eq.heater_fuel_type`
+    **Rule Assertion:**  
+    - CASE 1: if the fuel source is NATURAL_GAS, then PASS: `if heater_fuel_type == "NATURAL_GAS": PASS`
+    - Case 2: Else if the fuel source is propane: `elif: heater_fuel_type == "PROPANE": UNDETERMINED and raise_message "BASELINE SERVICE WATER HEATING EQUIPMENT SOURCE IS MODELED AS PROPANE. VERIFY IF NATURAL GAS IS NOT AVAILABLE FOR THE PROPOSED BUILDING SITE AS DETERMINED BY THE RATING AUTHORITY."`
+    - Case 3: All others fail: `else: FAIL`
+
+     **Rule Assertion - Zone:**
+    - Case1: rule_status is PASS: `if rule_status == "PASS": PASS`.
+    - Case2: rule_status is UNDETERMINED, UNDETERMINED & return rule_note: `elsif rule_status == "UNDETERMINED": UNDETERMINED; rule_note`
+    - Case3: rule_status is FAIL, FAIL & return rule_note: `elsif rule_status == "FAIL": FAIL; rule_note`
+
+
+**Notes:**
+
+1. original language specified gas storage water heaters only.  The rule applies only to gas storage water heaters because "storage" type water heaters are the only valid baseline fossil-fuel water heaters.  However, the heater type is checked in another rule.  Can we just check fuel-type here?
```

Comment:
Makes sense to me to evaluate for all non-electric water heaters regardless of whether it is a storage type